cluster/etcd-stack.yaml

```diff
@@ -144,9 +151,29 @@ Resources:
 {{- if eq .Cluster.ConfigItems.etcd_stack_specify_vpc_in_security_group "true"}}
       VpcId: "{{.Cluster.ConfigItems.vpc_id}}"
 {{- end }}
+{{- if eq .Cluster.ConfigItems.etcd_stack_security_group_temp "true"}}
+  EtcdSecurityGroupTemp:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Etcd Appliance Security Group
+      SecurityGroupIngress:
+      - IpProtocol: tcp
+        FromPort: 22
+        ToPort: 22
+        CidrIp: "{{.Values.vpc_ipv4_cidr}}"
+      - IpProtocol: tcp
+        FromPort: 2381
+        ToPort: 2381
+        CidrIp: "{{.Values.vpc_ipv4_cidr}}"
+      - IpProtocol: tcp
+        FromPort: 9100
+        ToPort: 9100
+        CidrIp: "{{.Values.vpc_ipv4_cidr}}"
+      VpcId: "{{.Cluster.ConfigItems.vpc_id}}"
       Tags:
         - Key: InfrastructureComponent
           Value: true
```

Comment:
nit: seems like the `Tags` is missing from the old security group with this diff. It should be on both